### DISCUSSION TITLE
category view in new update messy and ugly

### DISCUSSION THREAD

**Discussion by ppeyman** [breezefront/community#45] [5. General]
hi dear 
after update to version 2.1.0 evolution and blank theme to 2.1.0 we have add to cart button on mobile views chrome browser 
how can i fix it like old 

![image](https://user-images.githubusercontent.com/13271466/211744776-9efe450b-fab8-4903-a982-ba2c888b9ac6.png)

**Comment by vovayatsyuk** [comment]
This is the commit that made them visible: https://github.com/breezefront/theme-frontend-breeze-evolution/commit/a10df57a5de816a1e80429a65f613e4450fba237

To bring back the old styles, add the following variable to your custom theme:

```
@listing-grid-item-actions-primary__hover-none__position: false;
```
